README.md

```diff
@@ -1,72 +1,31 @@
-# ⚠️ THIS PROJECT IS NOT MAINTAINED ANYMORE ⚠️
+<img width=200px height=200px src="assets/readme_icon.svg"/>
 
-This project has been officially dropped due to lack of interest and political differences. If anyone is interested in continuing developement, feel free to fork it. For any questions you can message [krawieck](https://matrix.to/#/@krawieck:matrix.org) (who was responsible for the flutter app) or [shilangyu](https://matrix.to/#/@shilangyu:matrix.org) (who was responsible for lemmy_api_client).
+# Lemmynade
 
----
+A fork of the original lemmur app we all knew and loved. It was unfortunately abandoned but this project aims to keep the legacy it started.
 
-<div align="center">
+## How to Install
+- Currently there are no binary downloads available as this is just the start of this fork. However, you can reference the old readme [here](OLDREADME.md).
```

Comment:
There are now binaries to install in the releases tab. F-Droid, google play, and iOS are pending verification.

I updated the readme on master so you can just pull the screenshots and howto into that readme, then you can delete OLDREADME and the old privacy policy